arrow-core/src/main/kotlin/arrow/core/extensions/ordering/hash/OrderingHash.kt

```diff
@@ -30,5 +31,5 @@ fun Ordering.hashWithSalt(arg1: Int): Int = arrow.core.Ordering.hash().run {
   "UNCHECKED_CAST",
   "NOTHING_TO_INLINE"
 )
-@Deprecated("@extension kinded projected functions are deprecated", ReplaceWith("Hash.ordering()", "arrow.core.Hash", "arrow.core.ordering"))
+@Deprecated(OrderingDeprecation)
```

Comment:
Here should we use `HashDeprecation` or `OrderingDeprecation` ?

Reply:
It's both derpecated so.. 😅 I preferred `Ordering` here since the type in itself is deprecated, so by extension all behavior for it is also deprecated.

Reply:
perfect 👍 👍